src/ui/datetime/datetime.tsx

```diff
@@ -1,108 +1,108 @@
 import clsx from 'clsx';
 import { isEmpty, pick } from 'lodash';
 import type { SyntheticEvent } from 'react';
-import { Fragment, useEffect, useState } from 'react';
+import { Fragment, useMemo, useState } from 'react';
 import { z } from 'zod';
 
 import DateInput from '../date-input/date-input';
 
-const schema = z.object({
-  day: z.string({
-    required_error: 'Day is required',
-  }),
-  month: z.string({
-    required_error: 'Month is required',
-
-  }),
-  year: z.string({
-    required_error: 'Year is required',
-
-  }),
-  hour: z.string({
-    required_error: 'Hour is required',
-
-  }),
-  minute: z.string({
-    required_error: 'Minute is required',
+const dateSchema = z.object({
+  day: z.number({
+    invalid_type_error: 'Day is required',
+  }).min(1).max(31),
```

Comment:
This is not correct validation here, but maybe sufficient?
We have months shorter than 31

Reply:
It's too complicated to create correct validation for this. This feature is used only by admins and they are smart :D